### PR TITLE
Use githubRef to pass the branch name along to the CLI

### PR DESCRIPTION
**Pull request by jleven**
Must update the CLI before this can be released

### CODE REVIEW

File: src/action.js [hunk 1]
```diff
@@ -68,6 +70,8 @@ async function runCodeseeMapUpload(config, origin) {
     `https://github.com/${origin}`,
     "-a",
     config.apiToken,
+    "-b",
+    config.githubRef,
```
Comment (graysonarts) [inline src/action.js:74]: I'm assuming there is a related codesee-cli change for this?

Comment (jleven) [reply src/action.js:74]: https://github.com/Codesee-io/codesee/pull/788

File: action.yml [hunk 2]
```diff
@@ -10,6 +10,9 @@ inputs:
   support_typescript:
     description: Should tsconfig.json be honored when traversing dependencies
     default: false
+  github_ref:
+    description: The github ref (value of $GITHUB_REF) which references the branch name being processed
+    required: true
```
Comment (jleven) [inline action.yml:15]: @graysonarts -- should this be required? Will it break old installations?

File: src/action.js [hunk 3]
```diff
@@ -14,12 +14,14 @@ function getConfig() {
   const supportTypescript = core.getBooleanInput("support_typescript", {
     required: false,
   });
+  const githubRef = core.getInput("github_ref", { required: true });
```
Comment (jleven) [inline src/action.js:17]: Again, question about required vs optional

**Pull request merged**